cloud/user/user.go

```diff
@@ -67,3 +129,99 @@ func IsRoleValid(role string) error {
 	}
 	return ErrInvalidRole
 }
+
+func selectUser(users []astrocore.User) (astrocore.User, error) {
+	table := printutil.Table{
+		Padding:        []int{30, 50, 10, 50, 10, 10, 10},
+		DynamicPadding: true,
+		Header:         []string{"#", "FULLNAME", "EMAIL", "ID", "ORGANIZATION ROLE", "CREATE DATE"},
+	}
+
+	fmt.Println("\nPlease select the user who's role you would like to update:")
+
+	userMap := map[string]astrocore.User{}
+	for i := range users {
+		index := i + 1
+		table.AddRow([]string{
+			strconv.Itoa(index),
+			users[i].FullName,
+			users[i].Username,
+			users[i].Id,
+			*users[i].OrgRole,
```

Comment:
Why is this a pointer, since the value is just a constant string?

Reply:
because it's a pointer in API gen go